app/controllers/check.controller.js

```diff
@@ -5,10 +5,37 @@
  * @module CheckController
  */
 
+const DetermineSupplementaryBillingFlagsService = require('../services/import/determine-supplementary-billing-flags.service.js')
 const ProcessLicenceReturnLogsService = require('../services/jobs/return-logs/process-licence-return-logs.service.js')
 
 const redirectStatusCode = 204
 
+/**
+ * A test end point for the licence supplementary billing flags process
+ *
+ * This endpoint takes a licenceId and an expired, lapsed and revoked date. It passes this onto the
+ * `DetermineSupplementaryBillingFlagsService` to test if it correctly flags the licence for supplementary billing. This
+ * normally happens during the licence import process.
+ *
+ * @param request - the hapi request object
+ * @param h - the hapi response object
+ *
+ * @returns {Promise<object>} - A promise that resolves to an HTTP response object with a 204 status code
+ */
+async function flagForBilling (request, h) {
+  const { licenceId, expiredDate, lapsedDate, revokedDate } = request.payload
+
+  const transformedLicence = {
+    expiredDate,
+    lapsedDate,
+    revokedDate
+  }
+
+  DetermineSupplementaryBillingFlagsService.go(transformedLicence, licenceId)
```

Comment:
Does this need to be awaited? As this is for testing (we don't worry about hitting our 1 sec or less response times), plus we're just doing a single licence so should be fine anyway.

Probably best so the person testing doesn't expect the change has happened as soon as they get a response.